src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -159,8 +159,6 @@ public String getApiUri() {
         } catch (RateLimitExceededException rle) {
             listener.getLogger().format("%n%s%n%n", rle.getMessage());
             throw new InterruptedException();
-        } catch (IOException e) {
-            // may be a user... ok to ignore
```

Comment:
Well yes the problem was that `computeChildren` was not throwing an exception. But what was the point here where an exception _should_ have been thrown in what you reproduced? The call to `isCredentialValid` perhaps?